.travis.yml

```diff
@@ -3,18 +3,23 @@ sudo: required
 services:
   - docker
 
-env: SH="docker exec -t ubuntu-test bash -c"
+env:
+  matrix:
+  - ROS_VERSION=kinetic BUILD_TYPE=Debug
+  - ROS_VERSION=kinetic BUILD_TYPE=Release
+  - ROS_VERSION=melodic BUILD_TYPE=Debug
+  - ROS_VERSION=melodic BUILD_TYPE=Release
     
 before_install:
-  - docker build -t ubuntu ./docker
+  - docker build -t ubuntu --build-arg ROS_VERSION=$ROS_VERSION ./docker
   - docker run --name "ubuntu-test" -d -v $(pwd):/root/catkin_ws/src/arc_utilities -w /root/catkin_ws ubuntu tail -f /dev/null
   - docker ps
 
 install:
-  - $SH "./src/arc_utilities/docker/setup.sh"
+  - docker exec -ti ubuntu-test bash -c "./src/arc_utilities/docker/setup.sh $ROS_VERSION $BUILD_TYPE"
```

Comment:
Maybe add `rosversion -d`.

In the build output it would be nice to feel more confident about the ros version.
The environment variable is visible, but if only some builds fail I would want more confidence that the ros version is as expected
https://travis-ci.com/UM-ARM-Lab/arc_utilities/builds/107000935